tree/ntupleutil/v7/inc/ROOT/RNTupleInspector.hxx

```diff
@@ -53,15 +56,20 @@ auto file = TFile::Open("data.rntuple");
 auto rntuple = file->Get<RNTuple>("NTupleName");
 auto inspector = RNTupleInspector::Create(rntuple).Unwrap();
 
-std::cout << "The compression factor is " << std::fixed << std::setprecision(2)
-                                          << inspector->GetCompressionFactor()
-                                          << std::endl;
+std::cout << "The compression factor is " << inspector->GetCompressionFactor()
+          << " using compression settings " << inspector->GetCompressionSettings()
+          << std::endl;
```

Comment:
It might be worth it to have a tutorial to show more of the functionality. What do you think? This would be addressed by a separate PR>